plugin/src/main/java/org/jenkinsci/plugins/workflow/cps/CpsFlowExecution.java

```diff
@@ -632,11 +632,15 @@ private CpsScript parseScript() throws IOException {
             trusted = new CpsGroovyShellFactory(this).forTrusted().build();
             shell = new CpsGroovyShellFactory(this).withParent(trusted).build();
 
-            s = (CpsScript) shell.reparse("WorkflowScript",script);
+            s = (CpsScript) shell.reparse("WorkflowScript", script);
 
             for (Entry<String, String> e : loadedScripts.entrySet()) {
                 shell.reparse(e.getKey(), e.getValue());
             }
+        } catch (groovyjarjarasm.asm.MethodTooLargeException x) {
```

Comment:
I do not think we want to refer to this type statically. Maybe by `simpleName`.

Reply:
Updated PR to `import` the class and catch as shorter `MethodTooLargeException`. Is it OK so?